External/FEXCore/Source/Interface/HLE/Thunks/Thunks.cpp

```diff
@@ -139,7 +242,33 @@ namespace FEXCore {
                 // sha256(fex:allocate_host_trampoline_for_guest_function)
                 { 0x9b, 0xb2, 0xf4, 0xb4, 0x83, 0x7d, 0x28, 0x93, 0x40, 0xcb, 0xf4, 0x7a, 0x0b, 0x47, 0x85, 0x87, 0xf9, 0xbc, 0xb5, 0x27, 0xca, 0xa6, 0x93, 0xa5, 0xc0, 0x73, 0x27, 0x24, 0xae, 0xc8, 0xb8, 0x5a },
                 &AllocateHostTrampolineForGuestFunction
-            }
+            },
+            // VDSO
+            {
+                // sha256(libVDSO:time)
```

Comment:
Same question here - aren't those provided by `thunks.inl`?

(Apologies if this seems picky; the reason I'm asking is that I plan on unifying the generated files in the future, making inclusion of them non-optional)

Reply:
I think the issue there is that the vdso is implicitly loaded, so the onload entry points aren't run. We could solve that though.